### PR TITLE
[RHACS] Added release notes for 3.68.0

### PR DESCRIPTION
**Pull request by gaurav-nelson**
Release notes for upcoming release. Preview https://rel-notes-368.tiiny.site/

Applies to 3.68.0

<hr>

NOTE:

- This PR is for RHACS docs.
- It doesn't require any special labels or milestones.

### CODE REVIEW

File: release_notes/368-release-notes.adoc [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+== New features
+
+[id="vulnerability-triage-workflows"]
+=== Vulnerability triage workflows
+{product-title} 3.68 includes the ability to triage vulnerabilities in a variety of ways to support your vulnerability management process.
+See xref:../operating/manage-vulnerabilities.adoc#vulnerability-management-proccess_acs-operating-manage-vulnerabilities[Managing vulnerabilities] for more information.
+
+[id="report-scheduling"]
+=== Report scheduling for vulnerabilities
+{product-title} 3.68 includes the ability to schedule reports for vulnerabilities helping you to send scheduled communications to key stakeholders to assist in the vulnerability management process. See xref:../operating/manage-vulnerabilities.adoc#vulnerability-management-reporting_acs-operating-manage-vulnerabilities[Reporting vulnerabilities to teams] for more information.
```
Comment (kalexand-rh) [inline release_notes/368-release-notes.adoc:22]:
```suggestion
{product-title} 3.68 includes the ability to schedule reports for vulnerabilities which helps you send scheduled communications to key stakeholders to assist in the vulnerability management process. See xref:../operating/manage-vulnerabilities.adoc#vulnerability-management-reporting_acs-operating-manage-vulnerabilities[Reporting vulnerabilities to teams] for more information.
```

File: release_notes/368-release-notes.adoc [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+[id="use-aws-ecr-assumeroles"]
+=== Use AWS ECR AssumeRoles
+AWS AssumeRoles allows you to define roles with specific permissions and then granting users access to those roles. {product-title} 3.68 includes the ability to use AWS ECR AssumeRoles to configure roles and grant various levels of access to users.
```
Comment (kalexand-rh) [inline release_notes/368-release-notes.adoc:26]: AWS AssumeRoles or AWS ECR AssumeRoles or are they really interchangeable?

File: release_notes/368-release-notes.adoc [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+=== Enhancements for CI outputs
+Red Hat has improved the usability of {product-title} CI integrations. CI outputs now show additional detailed information about the vulnerabilities and the security policies responsible for broken builds.
+
+[id="inactive-software-components"]
+=== Inactive Software Component Identification
+Red Hat has improved the ability of users to identify if a software package that is being used inside of a container image is inactive, which helps users to triage if a software package may be able to be removed as a hardening step or vulnerability remediation.
+
+[id="known-issue"]
+== Known issue
+When the same software package is installed in multiple layers in a docker image {product-title-short} may incorrectly identify the state of that package.
```
Comment (kalexand-rh) [inline release_notes/368-release-notes.adoc:39]:
```suggestion
When the same software package is installed in multiple layers in a docker image, {product-title-short} might incorrectly identify the state of that package.
```

File: release_notes/368-release-notes.adoc [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+
+[id="important-bug-fixes"]
+== Important bug fixes
+
+* *ROX-8709*: Previously, searching for CVE's with a specific severity did not returned any results. This issue has been fixed.
+* *ROX-8983*: Previously, when configuring the *Manage Watches* feature, if you added more than 12 images to the watch list, the image list would not display properly. This issue has been fixed.
```
Comment (kalexand-rh) [inline release_notes/368-release-notes.adoc:45]:
```suggestion
* *ROX-8983*: Previously, when configuring the *Manage Watches* feature, if you added more than 12 images to the watch list, the image list did not display properly. This issue has been fixed.
```

File: release_notes/368-release-notes.adoc [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+* *ROX-8276*: Previously, when the {product-title-short} Operator accessed the `central-htpasswd` secret, it would create a false positive policy violation for the *OpenShift: Advanced Cluster Security Central Admin Secret Accessed* default policy. This issue has been fixed.
```
Comment (kalexand-rh) [inline release_notes/368-release-notes.adoc:46]:
```suggestion
* *ROX-8276*: Previously, when the {product-title-short} Operator accessed the `central-htpasswd` secret, it created a false positive policy violation for the *OpenShift: Advanced Cluster Security Central Admin Secret Accessed* default policy. This issue has been fixed.
```

File: release_notes/368-release-notes.adoc [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+
+[id="security-update"]
+== Security update
+In earlier versions of {product-title}, the `write` permission for the `APIToken` resource allowed users to create API tokens for any role, including the `admin` role. This issue has been fixed.
```
Comment (kalexand-rh) [inline release_notes/368-release-notes.adoc:50]: Is it just the admin role you're removing the ability to change? If so, consider:
```suggestion
In earlier versions of {product-title}, the `write` permission for the `APIToken` resource allowed users to create API tokens for any role, including the `admin` role. This issue has been fixed, and it is no longer possible to create API tokens for the `admin` role..
```

File: release_notes/368-release-notes.adoc [hunk 7]
```diff
@@ -0,0 +1,137 @@
...
+
+[id="important-system-changes"]
+== Important system changes
+
+* {product-title-short} 3.68 includes updates for the `Log4Shell` vulnerability detection policy. With this update this policy also detects CVE-2021-45046 and it includes the updated remediation based on the latest guidance by the Apache Logging security team.
```
Comment (kalexand-rh) [inline release_notes/368-release-notes.adoc:55]:
```suggestion
* {product-title-short} 3.68 includes updates for the `Log4Shell` vulnerability detection policy. With this update, this policy also detects CVE-2021-45046, and it includes the updated remediation based on the latest guidance by the Apache Logging security team.
```

File: release_notes/368-release-notes.adoc [hunk 8]
```diff
@@ -0,0 +1,137 @@
...
+* Before this release, CVEs could be snoozed using global write access on the `Images` resource. Beginning with {product-title-short} 3.68:
```
Comment (kalexand-rh) [inline release_notes/368-release-notes.adoc:56]:
```suggestion
* Before this release, only global write access on the `Images` resource was necessary to snooze CVEs . Beginning with {product-title-short} 3.68:
```

File: release_notes/368-release-notes.adoc [hunk 9]
```diff
@@ -0,0 +1,137 @@
...
+Red Hat has improved the ability of users to identify if a software package that is being used inside of a container image is inactive, which helps users to triage if a software package may be able to be removed as a hardening step or vulnerability remediation.
+
+[id="known-issue"]
+== Known issue
+When the same software package is installed in multiple layers in a docker image {product-title-short} may incorrectly identify the state of that package.
+
+[id="important-bug-fixes"]
+== Important bug fixes
+
+* *ROX-8709*: Previously, searching for CVE's with a specific severity did not returned any results. This issue has been fixed.
```
Comment (kalexand-rh) [inline release_notes/368-release-notes.adoc:44]:
```suggestion
* *ROX-8709*: Previously, searching for CVEs with a specific severity did not returned any results. This issue has been fixed.
```

File: release_notes/368-release-notes.adoc [hunk 10]
```diff
@@ -0,0 +1,137 @@
...
+== Important system changes
+
+* {product-title-short} 3.68 includes updates for the `Log4Shell` vulnerability detection policy. With this update this policy also detects CVE-2021-45046 and it includes the updated remediation based on the latest guidance by the Apache Logging security team.
+* Before this release, CVEs could be snoozed using global write access on the `Images` resource. Beginning with {product-title-short} 3.68:
+** To snooze CVEs, you must have `write` permission for the `VulnerabilityManagementRequests` resource.
+** To approve requests, you must have `write` permission for the `VulnerabilityManagementApprovals` resource.
+* When you upgrade to {product-title-short} 3.68, roles that include `write` access on the `Images` resource will have `write` permissions for both  `VulnerabilityManagementRequests` and `VulnerabilityManagementApprovals` resource.
+Red Hat recommends updating the roles to only include the least amount of resources required for each role.
+* If you have installed {product-title} using Helm, this update disabled the cluster configuration options in the {product-title-short} portal. You can continue to use Helm configuration files.
```
Comment (kalexand-rh) [inline release_notes/368-release-notes.adoc:61]:
```suggestion
* If you installed {product-title} using Helm, this update disabled the cluster configuration options in the {product-title-short} portal. You can continue to use Helm configuration files.
```

File: release_notes/368-release-notes.adoc [hunk 11]
```diff
@@ -0,0 +1,137 @@
...
+====
+* Red Hat has moved the following images to new repositories:
++
+[%header,cols="1,2,2"]
+|===
+|Image
+|Old repository
+|New repository
+
+|main
```
Comment (kalexand-rh) [inline release_notes/368-release-notes.adoc:76]: Please double-check the [markup chart](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference). The way I read it, all the repo names and the image names should be inclosed in backticks.

**Review by kalexand-rh — COMMENTED** [review]
I don't see a preview link, so all comments are based on the diff. I'm happy to take a look at the rendered file if you have a preview available.

**Comment by gaurav-nelson** [comment]
@kalexand-rh Can you please check again, preview at https://rel-notes-368.tiiny.site/ (PS: only a single page).

File: release_notes/368-release-notes.adoc (hunk 8 above)
Comment (gaurav-nelson) [reply release_notes/368-release-notes.adoc]: I've updated the description.

**Comment by gaurav-nelson** [comment]
Merging this PR, if there are changes I'll do a followup.

**Pull request merged**